examples/GroupCharacters.jl

```diff
@@ -487,16 +498,19 @@ function group_class_function(tbl::GAPGroupCharacterTable, values::Vector{QabMod
     return GAPGroupClassFunction(tbl, GAP.Globals.ClassFunction(tbl.GAPTable, GAP.julia_to_gap([gap_cyclotomic(x) for x in values])))
 end
 
-#function group_class_function(G::Oscar.GAPGroup, values::Vector{QabModule.QabElem})
-#end
+function group_class_function(G::Oscar.GAPGroup, values::Vector{QabModule.QabElem})
+    return group_class_function(character_table(G), values)
+end
 
 function trivial_character(tbl::GAPGroupCharacterTable)
     val = QabModule.QabElem(1)
     return group_class_function(tbl, [val for i in 1:ncols(tbl)])
 end
 
-#function trivial_character(G::Oscar.GAPGroup)
-#end
+function trivial_character(G::Oscar.GAPGroup)
+    val = QabModule.QabElem(1)
+    return group_class_function(G, [val for i in 1:GAP.Globals.NrConjugacyClasses(G.X)])
```

Comment:
Maybe
```suggestion
    return group_class_function(G, fill(val, GAP.Globals.NrConjugacyClasses(G.X))
```
?

Reply:
can be done, but as it should not block the pull request, I merged